source/administration/changelog.md

```diff
@@ -178,7 +178,7 @@ Thank you to everyone who contributed to the Mattermost project in March 2019!
 
-## Release v5.9
+## Release v5.9 (`ESR <https://docs.mattermost.com/administration/extended-support-release.html?>`_)
```

Comment:
Not 100% sure if this and the below link formatting works properly.

Otherwise this PR looks good.

Reply:
Thanks - I can review -  (I couldn't get the preview to work.)